pyproject.toml

```diff
@@ -13,6 +13,9 @@ python = "^3.7.1"
 thrift = "^0.13.0"
 pyarrow = "^5.0.0"
 pandas = "^1.3.0"
+click=">=7.0"
```

Comment:
I recommend against depending on click in a non-CLI library.

Reply:
thanks addressed, replaced with logger